docs/_docs/hivemq/schema.md

```diff
@@ -1,17 +1,17 @@
 ---
 nav_order: 1
-redirect_from: /docs/hivemq/schemas.html
+redirect_from: /docs/hivemq/schema.html
```

Comment:
Maybe preserve the redirect of "/docs/hivemq/schemas.html" in addition, so that existing links do not break.

Reply:
What is the syntax for multiple redirects?

Reply:
```
redirect_from:
  - /post/123456789/
  - /post/123456789/my-amazing-post/
```

https://github.com/jekyll/jekyll-redirect-from